arch/arm/src/armv8-r/Toolchain.defs

```diff
@@ -23,10 +23,10 @@ ARCHCPUFLAGS += -march=armv8-r
 ifeq ($(CONFIG_ARCH_FPU),y)
   LLVM_ABITYPE := eabihf
 
-  ifeq ($(CONFIG_ARCH_CORTEXR52)$(CONFIG_ARM_NEON),yy)
```

Comment:
The commit message and these changes are now in conflict with the PR topic. Modifications to armv8 and risc-v but the topic is for risc-v only.

Please restore the original patch and remove the armv8 stuff

Reply:
ok

Reply:
Thanks, LGTM now